test/ng/directive/ngModelSpec.js

```diff
@@ -847,6 +847,38 @@ describe('ngModel', function() {
         expect(ctrl.$valid).toBe(true);
       });
 
+      it('should treat all responses as boolean for synchronous validators', function() {
+        var curry = function(v) {
```

Comment:
Interesting, I'll make the switch. I was actually hoping to talk about the `curry()` anyways, because it's now used three times in this file (hence why I used it in the first place). I'll submit a separate PR to fix the others.